cms/models.py

```diff
@@ -676,36 +683,41 @@ def serve(self, request, *args, **kwargs):
         return super().serve(request, *args, **kwargs)
 
     def process_form_submission(self, form):
+
+        try:
+            converted_income = determine_income_usd(
+                float(form.cleaned_data["your_income"]),
+                form.cleaned_data["income_currency"],
+            )
+        except NotSupportedException:
+            raise ValidationError("Currency not supported")
+
+        courseware = self.get_parent_product()
+        income_usd = round(converted_income, 2)
+        tier = determine_tier_courseware(courseware, income_usd)
```

Comment:
This ends up delivering an error message to the learner if there's no tier, which is undesirable. We'd still want to capture the request so that it can be processed manually. Can you wrap this in a try/except block and leave it in `PENDING_MANUAL_APPROVAL` if there's no tier? It looks like the tier FK in FlexiblePrice is already nullable.

Reply:
In my opinion, there is no need to apply for the flexible pricing if we don't have any tier. @pdpinch / @rhysyngsun what's your opinion on that?

Reply:
This is fine as is.